frame/wormhole/issuing/s2s/src/lib.rs

```diff
@@ -41,7 +41,7 @@ use frame_support::{
 	transactional, PalletId,
 };
 use frame_system::ensure_signed;
-use sp_runtime::{traits::Convert, DispatchError, MultiSignature, MultiSigner};
+use sp_runtime::{traits::Convert, DispatchError};
```

Comment:
Please update the `#[pallet::weight(<T as Config>::WeightInfo::issue_from_remote().saturating_add(2_000_000 * 2))]`.